### DISCUSSION TITLE
How do I actually source this after installing to enable command `:Rocks`?

### DISCUSSION THREAD

**Discussion by hacker-DOM** [lumen-oss/rocks.nvim#477] [Q&A]
The snippet I copied into init.lua only adds to runtimepath, but how do I actually enable `:Rocks` command? I'm using Lazy fwiw. Also tried `:lua require('rocks')` (doesn't do anything), `:lua require('rocks.nvim')` (can't be found).

**Comment by mrcjkb** [comment]
Hey :wave: 

rocks.nvim registers the `:Rocks` command automatically.
If you have used lazy.nvim to install rocks.nvim, perhaps it's being lazy-loaded?
